### PR TITLE
Improve `python-test.yml`

### PR DESCRIPTION
**Pull request by sami-m-g**
:heart: Thank you for your contribution!

### Description

> included new commits to existing PRs
> removed creating db in pipeline

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://github.com/sentier-dev.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] I'm aware of the [contributing guidelines](https://github.com/sentier-dev/sentier.dev/blob/main/Engineering/CONTRIBUTING.md).
- [ ] I've created [logical separate commits](https://github.com/sentier-dev/sentier.dev/blob/main/Engineering/CONTRIBUTING.md#git-commits) and followed the [commit message format](https://github.com/sentier-dev/sentier.dev/blob/main/Engineering/CONTRIBUTING.md#git-commits).
- [ ] If this PR resolves or addresses an issue, I've linked it in the [commit message](https://github.com/sentier-dev/sentier.dev/blob/main/Engineering/CONTRIBUTING.md#git-commits).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added third-party code (copy/pasted or new dependencies), please make sure this has been discussed in a github issue.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: .github/workflows/python-test.yml [hunk 1]
```diff
@@ -40,7 +40,6 @@ jobs:
       postgres:
         image: postgres
         env:
-          POSTGRES_DB: testdb
```
Comment (jsvgoncalves) [inline .github/workflows/python-test.yml:44]: Why are we removing this exactly and what's the consequence?

Comment (sami-m-g) [reply .github/workflows/python-test.yml:44]: @jsvgoncalves this resolves the unit tests assuming a fresh install where we haven't created the database yet.

Comment (jsvgoncalves) [reply .github/workflows/python-test.yml:44]: Still not clear what the consequence of removing the DB name from the env vars is.

Comment (sami-m-g) [reply .github/workflows/python-test.yml:44]: @jsvgoncalves the postgresql server is created before running the unit tests but the `testdb` database is not. The unit tests then test multiple scenarios including the fresh install where there is no created database.

**Pull request merged**